README.md

```diff
@@ -1 +1,30 @@
-# portfolio-1-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Description of Portfolio.
+This is the "Portfolio" Project with just Html and Css
+
+## Built With
+
+Html and Css
+
+
+## Getting Started
+
+npm clone https://github.com/heintayzar-hm/portfolio-1-microverse.git
+
+
```

Comment:
Great job @heintayzar-hm 🤝

- [ ] Kindly add your social handles to your readme file and make sure they are links directing the user to either your `GitHub Profile`, your `LinkedIn`, and your `Twitter Profile`.